xrayvision/visibility.py

```diff
@@ -352,6 +352,38 @@ def __repr__(self):
         """
         return f"{self.__class__.__name__}< {self.u.size}, {self.visibilities}>"
 
+    def __eq__(self, other):
+        """
+        Checks whether two VisMeta objects are equal.
+
+        Does not check whether their metas are equal.
```

Comment:
So this is really checking `avis.data == bvis.data` but we don't have a `.data`? What is the rational for not checking the `meta`.

Reply:
The thought is that even if the metadata is different, the visibilities themselves could be the same.  For example, if the source is invariant over time or energy, then the visibilities should be the same, even it the metadata is different.

Reply:
I'm just trying to think of other python, sunpy or astropy etc object that work this can see arguments both ways - I guess it not blocker at this point though.

Reply:
Ok so the only two place I can see something similar do opposite things `sunpy.timeseries` (https://github.com/sunpy/sunpy/blob/50f4588b53b1d575623dfc8a2536595e76261746/sunpy/timeseries/timeseriesbase.py#L870) compares the meta where as `astropy.table` (https://github.com/astropy/astropy/blob/8c21018c0f0d9c7de38ed4f30845f9ae0f825b94/astropy/table/table.py#L3818) only compares the rows. So again once we document think all good.